2024-02-18 20:57:40 - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "c:\Users\ellie\OneDrive\Documents\Programming\Python\Level 6\Bookerplay\.venv\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\ellie\OneDrive\Documents\Programming\Python\Level 6\Bookerplay\.venv\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\ellie\OneDrive\Documents\Programming\Python\Level 6\Bookerplay\.venv\Lib\site-packages\flask\app.py", line 868, in full_dispatch_request
    rv = self.preprocess_request()
         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\ellie\OneDrive\Documents\Programming\Python\Level 6\Bookerplay\.venv\Lib\site-packages\flask\app.py", line 1243, in preprocess_request
    rv = self.ensure_sync(before_func)()
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ellie\OneDrive\Documents\Programming\Python\Level 6\Bookerplay\app\user\routes.py", line 14, in make_session_permanent
    current_app.permanent_session_lifetime = timedelta(minute=5)
                                             ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
2024-02-18 20:58:23 - INFO - Username: admin logged in successfully
2024-02-18 20:58:25 - INFO - Username: admin accessed reservations
2024-02-18 20:58:35 - INFO - Username: admin logged out
